### PR TITLE
app-admin/xstow: Bumped to version xstow-1.0.2

### PR DESCRIPTION
**Pull request by majorkingleo**
update to EAP7, fixes EPREFIX glitches

Package-Manager: Portage-2.3.67-prefix, Repoman-2.3.16

The patch is copied from previous version, the ebuild files too (and fixed EAPI7 stuff).

### CODE REVIEW

File: app-admin/xstow/xstow-1.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
```
Comment (thesamesam) [inline app-admin/xstow/xstow-1.0.2.ebuild:6]: We can drop eutils

File: app-admin/xstow/xstow-1.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="ncurses"
+
+DEPEND="ncurses? ( sys-libs/ncurses:0= )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	eapply "${FILESDIR}"/${P}-ncurses.patch
```
Comment (thesamesam) [inline app-admin/xstow/xstow-1.0.2.ebuild:21]: We usually define PATCHES=() in global scope instead (above src_prepare)

PATCHES=(
    "${FILESDIR}"/${P}-ncurses.patch
)

File: app-admin/xstow/xstow-1.0.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	default
+}
+
+src_configure() {
+	econf $(use_with ncurses curses)
+	default
+}
+
+src_install() {
+	emake DESTDIR="${D}" docdir="${EPREFIX}""/usr/share/doc/${PF}/html" install
```
Comment (thesamesam) [inline app-admin/xstow/xstow-1.0.2.ebuild:32]:
```suggestion
	emake DESTDIR="${D}" docdir="${EPREFIX}"/usr/share/doc/${PF}/html install
```

**Review by thesamesam — COMMENTED** [review]
Could you change some minor bits -- nothing major?

Also, could you add the GCO sign off (signed-off-by) to your commit?

File: app-admin/xstow/xstow-1.0.2.ebuild (hunk 1 above)
Comment (majorkingleo) [reply app-admin/xstow/xstow-1.0.2.ebuild]: done

File: app-admin/xstow/xstow-1.0.2.ebuild (hunk 2 above)
Comment (majorkingleo) [reply app-admin/xstow/xstow-1.0.2.ebuild]: done

File: app-admin/xstow/xstow-1.0.2.ebuild (hunk 3 above)
Comment (majorkingleo) [reply app-admin/xstow/xstow-1.0.2.ebuild]: done

**Comment by majorkingleo** [comment]
So, is there anything left todo?

**Comment by thesamesam** [comment]
Sign-off still seems to be missing from the original commit. Could you squash them all, include it in the original, then I'll merge?

BTW:
```Package-Manager: Portage-2.3.67-prefix, Repoman-2.3.16```

This seems quite out of date.. maybe emerge --sync && emerge -a -uvDU && emerge -acv

**Comment by thesamesam** [comment]
Thanks, merged the newer #20255.

**Pull request closed without merging**